packages/elements/link/src/getLinkOnKeyDown.ts

```diff
@@ -0,0 +1,19 @@
+import { getPlatePluginOptions, KeyboardHandler } from '@udecode/plate-core';
+import isHotkey from 'is-hotkey';
+import { ELEMENT_LINK } from './defaults';
+import { getAndUpsertLink } from './transforms';
+import { WithLinkOptions } from './types';
+
+export const getLinkOnKeyDown = (
+  options?: WithLinkOptions
+): KeyboardHandler => (editor) => (e) => {
+  const { hotkey } = getPlatePluginOptions(editor, ELEMENT_LINK);
+
+  if (!hotkey) return;
+
+  if (isHotkey(hotkey, e as any)) {
+    e.preventDefault();
+
```

Comment:
We could call `e.stopPropagation()` to avoid docs search trigger?

Reply:
Agreed; added and tested this in 8c1da12.